examples/gp.py

```diff
@@ -54,7 +54,7 @@ def run_inference(model, args, rng, X, Y):
     init_params, potential_fn, constrain_fn = initialize_model(rng, model, X, Y)
     start = time.time()
     samples = mcmc(args.num_warmup, args.num_samples, init_params, num_chains=args.num_chains,
-                   sampler='hmc', potential_fn=potential_fn, constrain_fn=constrain_fn, progbar=None)
+                   sampler='hmc', potential_fn=potential_fn, constrain_fn=constrain_fn)
```

Comment:
@fehiepsi - Since your change disables progress bar with pmap anyways, I have kept the progress bar for the default case so that it is easy to track progress and users aren't confused. e.g. on the GPU, this will be slow for now, and it will be nice to see progress, otherwise users might think that it isn't making any progress.

Reply:
Thanks @neerajprad ! It is a legacy flag which I used to benchmark result.